script/publishing-api-sync-checks/case_study_sync_check.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../", File.dirname(__FILE__))
+
+require 'pathname'
+require 'logger'
+logger = Logger.new(STDERR)
+logger.info "Booting rails..."
+require '../../config/environment'
+logger.info "Booted"
+
+case_studies = CaseStudy.latest_published_edition
+check = DataHygiene::PublishingApiSyncCheck.new(case_studies)
+check.add_expectation("format") do |content_store_payload, model|
```

Comment:
Nitpicky, but I think `record` makes more sense for a variable name to be explicit that this is an instance of a model rather than the model itself.